extension/chrome.manifest

```diff
@@ -1,11 +1,3 @@
-overlay chrome://browser/content/browser.xul                chrome://personas/content/personas.xul    application={ec8030f7-c20a-464f-9b0e-13a3a9e97384}
-overlay chrome://messenger/content/messenger.xul            chrome://personas/content/messenger.xul   application={3550f703-e582-4d05-9a08-453d09bdfdc6}
-
-# Register a resource: protocol alias so we can refer to files in this package
-# with URLs of the form resource://personas/path/to/file,
-# f.e. resource://personas/modules/service.js.
-resource personas   ./
```

Comment:
Because resource protocol is registered by bootstrap.js, I removed it from chrome.manifest not to create conflicts.